packages/cli/src/config/schema.ts

```diff
@@ -846,6 +846,18 @@ export const schema = {
 					default: '',
 					env: 'N8N_UM_EMAIL_TEMPLATES_PWRESET',
 				},
+				workflowShared: {
+					doc: 'Overrides default HTML template for notifying that a workflow was shared (use full path)',
+					format: String,
+					default: '',
+					env: 'N8N_UM_EMAIL_TEMPLATES_WORKFLOW_SHARED',
+				},
+				credentialsShared: {
+					doc: 'Overrides default HTML template for notifying that credentials were shared (use full path)',
+					format: String,
+					default: '',
+					env: 'N8N_UM_EMAIL_TEMPLATES_CREDENTIALS_SHARED',
+				},
```

Comment:
I've always wondered why we do it this way. can we not ask users to overwrite the files by using docker volumes, or even extending the official images with custom templates.
I believe that the burden of customization related code should lie on the party making those customizations.

Reply:
@maspio Do you happen to know if embed customers usually need to customize email templates?